test/marathon_client_test.exs

```diff
@@ -21,13 +21,41 @@ defmodule MarathonEventExporter.MarathonClientTest do
     {:ok, fm} = start_supervised(FakeMarathon)
     base_url = FakeMarathon.base_url(fm)
     {:ok, _} = MarathonClient.stream_events(base_url, [self()])
+    # FIXME: stream_events returns as soon as the request has been sent, which
+    # means the server may not have received it yet. This creates a race
+    # between the stream setup and the first event, so until that's fixed we'll
+    # just wait a few milliseconds here.
+    Process.sleep(65)
```

Comment:
`65` seems like an odd number?

Reply:
Of course. It would have to be divisible by two to be even... :trollface: 

I carefully calculated the optimal value to use by asking the person next to me "quick, give me a number between ten and a hundred" and then checking that their response made the tests pass consistently about a dozen times.